data/json/items/armor/pets_dog_armor.json

```diff
@@ -0,0 +1,120 @@
+[
+  {
+    "type": "PET_ARMOR",
+    "id": "kevlar_harness",
+    "symbol": "[",
+    "color": "blue",
+    "name": "Kevlar dog harness",
+    "name_plural": "Kevlar dog harnesses",
+    "description": "A blue bulletproof harness, designed to be worn by canines in the service of local law enforcement.  You could put this on a friendly dog.",
+    "price": 30000,
+    "price_postapoc": 3000,
+    "material": [ "kevlar" ],
+    "weight": 4290,
+    "volume": 18,
+    "bashing": 5,
+    "to_hit": -1,
+    "flags": [ "IS_PET_ARMOR" ],
+    "material_thickness": 2,
+    "storage": 50,
+    "max_pet_vol": "35000 ml",
+    "min_pet_vol": "25000 ml",
+    "pet_bodytype": "dog"
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "acidchitin_harness_dog",
+    "copy-from": "kevlar_harness",
+    "color": "green",
+    "name": "biosilicified chitin dog mesh harness",
+    "name_plural": "biosilicified chitin dog mesh harnesses",
+    "description": "A makeshift harness of biosilicified chitin fitted to a thin mesh to be worn as protection for canines.  You could put this on a friendly dog.",
+    "price": 50000,
+    "price_postapoc": 5000,
+    "material": [ "acidchitin" ],
+    "weight": 5362,
+    "environmental_protection": 7
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "chitin_harness_dog",
+    "copy-from": "acidchitin_harness_dog",
+    "name": "chitin dog mesh harness",
+    "name_plural": "chitin dog mesh harnesses",
+    "description": "A makeshift harness of chitin fitted to a thin mesh to be worn as protection for canines.  You could put this on a friendly dog.",
+    "price": 35000,
+    "price_postapoc": 3500,
+    "material": [ "chitin" ],
+    "environmental_protection": 4
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "chainmail_harness_dog",
+    "copy-from": "kevlar_harness",
+    "color": "light_red",
+    "name": "chainmail dog coat",
+    "description": "Protecteth yon hund fram ful daemons!  You could put this on a friendly dog.",
+    "//": "Description is reference to Churl scenario text - 'protect your dog from foul undead'.",
+    "price": 25000,
+    "price_postapoc": 2500,
+    "material": [ "iron", "budget_steel" ],
+    "weight": 6434,
+    "material_thickness": 0
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "leather_harness_dog",
+    "copy-from": "kevlar_harness",
+    "color": "brown",
+    "name": "leather dog harness",
+    "name_plural": "leather dog harnesses",
+    "description": "A harness made from leather that can be attached to a canine for protection.  You could put this on a friendly dog.",
+    "price": 17500,
+    "price_postapoc": 1750,
+    "material": [ "cotton", "leather" ],
+    "weight": 2145,
+    "flags": [ "NO_SALVAGE" ]
```

Comment:
It is missing `IS_PET_ARMOR` flag.

Reply:
Good catch, I forgot to re-check if this work after adding them (I quickly added them when I noticed butchering gave inappropriate amounts but assumed still worked). Also, totally forgot about using extend. Fixed!